memory/table.go

```diff
@@ -492,20 +492,23 @@ func rowsAreEqual(ctx *sql.Context, schema sql.Schema, left, right sql.Row) (boo
 }
 
 // PeekNextAutoIncrementValue peeks at the next AUTO_INCREMENT value
-func (t *Table) PeekNextAutoIncrementValue(*sql.Context) (interface{}, error) {
```

Comment:
Problem is this could be a float. If you go this route you need affordances for that

Reply:
Same comment throughout